package.json

```diff
@@ -4,7 +4,7 @@
     "test": "meteor test --once --driver-package dispatch:mocha-phantomjs",
     "test-watch": "meteor test --driver-package practicalmeteor:mocha",
     "test-app-watch": "meteor test --full-app --driver-package practicalmeteor:mocha",
-    "lint": "eslint .",
+    "lint": "meteor node node_modules/eslint/bin/eslint.js .",
```

Comment:
I know, but it's not necessary. `npm run` defaults to executables in `node_modules/.bin`